src/keycloak/chart/values.yaml

```diff
@@ -15,12 +15,20 @@ domain: "###ZARF_VAR_DOMAIN###"
 # The primary Keycloak realm
 realm: uds
 
-# UDS Identity Config Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#templated-realm-values
-realmInitEnv:
-  GOOGLE_IDP_ENABLED: false
-  # Other UDS Identity Config fields that will be used in the realm.json initalization of keycloak
-  # GOOGLE_IDP_CLIENTID: ""
-  # GOOGLE_IDP_CLIENT_SECRET: ""
+# UDS Identity Config Realm Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#templated-realm-values
+# realmInitEnv:
```

Comment:
nit: I'm a fan of `realmInitEnv: {}` (not commented out, but empty by default) in addition to the commented block here. Shouldn't change any templating/functionality but draws more attention to the value existing imo.